.eslintrc.json

```diff
@@ -26,13 +26,49 @@
       "plugins": ["@typescript-eslint"],
       "parser": "@typescript-eslint/parser",
       "files": ["**/*.ts"],
+      "excludedFiles": ["**/*.test.ts", "*.js"],
+      "extends": [
+        "plugin:@typescript-eslint/eslint-recommended",
+        "plugin:@typescript-eslint/recommended",
+        "plugin:prettier/recommended"
+      ],
+      "rules": {
+        "@typescript-eslint/no-use-before-define": 0,
+        "class-methods-use-this": 0,
+        "no-useless-constructor": 0,
+        "@typescript-eslint/no-unused-vars": [
+          1,
+          {
+            "argsIgnorePattern": "res|next|^err|_",
+            "ignoreRestSiblings": true
+          }
+        ],
+        "@typescript-eslint/semi": 0,
+        "import/no-unresolved": "error",
+        "prettier/prettier": [
+          "error",
+          {
+            "trailingComma": "all",
+            "singleQuote": true,
+            "printWidth": 120,
+            "semi": false
+          }
+        ]
+      }
+    },
+    {
+      "plugins": ["@typescript-eslint"],
+      "parser": "@typescript-eslint/parser",
+      "files": ["**/*.test.ts"],
       "excludedFiles": "*.js",
       "extends": [
         "plugin:@typescript-eslint/eslint-recommended",
         "plugin:@typescript-eslint/recommended",
         "plugin:prettier/recommended"
       ],
       "rules": {
+        "dot-notation": "off",
```

Comment:
I am happy with this, was there a use case that convinced you to turn this off? We can always challenge the use of "any types" during the PR process to minimise it.

Reply:
This change is just for `*.test.ts` files as those rules specifically were preventing me from spying on private methods.
For application code, I think it's good to avoid using `any` still and those rules are still in place.